flexmeasures/api/common/utils/deprecation_utils.py

```diff
@@ -1,12 +1,76 @@
 from __future__ import annotations
 
-from flask import current_app, request, Blueprint, Response
+from flask import current_app, request, Blueprint, Response, after_this_request
 from flask_security.core import current_user
 import pandas as pd
 
 from flexmeasures.utils.time_utils import to_http_time
 
 
+def deprecate_field(
```

Comment:
Should we rename this to `deprecate_fields` (plural)?

Reply:
My hope is that the more typical use case will be to deprecate a single field, but sure.